PowerEditor/src/localization.h

```diff
@@ -45,7 +45,7 @@ class NativeLangSpeaker {
 	TiXmlNodeA * searchDlgNode(TiXmlNodeA *node, const char *dlgTagName);
 	bool changeDlgLang(HWND hDlg, const char *dlgTagName, char *title = NULL, size_t titleMaxSize = 0);
 	void changeLangTabDropContextMenu(HMENU hCM);
-	generic_string getSpecialMenuEntryName(const char *entryName) const;
+	generic_string getSpecialMenuEntryName(const char *entryName, const char *attrName = "menuId", const char *nodeName = "Entries") const;

```

Comment:
This function was coded with the old XML structure of english.xml. So we can
1. Remove the added parameters
2. Rename `entryName` to `nodeName`
3. Rename `getSpecialMenuEntryName` to `getSubMenuEntryName`
4. ...

Reply:
@donho 

I extended this function to work for any node in `<main>` (because such type doesn't exist), but if you want only for `<SubEntries>` then could be simplified.

Reply:
@ArkadiuszMichalski 
Yes, I did understand your intention. But this function is called only for recent files entry and I don't see the need for the other menu entry, so let's make it straight and simple.